winlogbeat/sys/strings_test.go

```diff
@@ -0,0 +1,77 @@
+package sys
+
+import (
+	"bytes"
+	"encoding/binary"
+	"testing"
+	"unicode/utf16"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func toUTF16Bytes(in string) []byte {
+	var u16 []uint16 = utf16.Encode([]rune(in))
```

Comment:
The `UTF16BytesToString()` function gets used for a `[]byte` -> `string` conversion and there aren't any functions in the stdlib that can do that directly (there's just `[]uint16` -> `string`).